### DISCUSSION TITLE
Permissions for user menu

### DISCUSSION THREAD

**Discussion by Alexcei88** [keycloak/keycloak#11738] [Archive]
There is a tab "Permissions" in old UI for Users menu if admin-fine-grained-authz feature is turned on.
On new UI I cannot find such tab although the feature is turned on too. 
 I want to flexibly configure user management in new UI as I can do in old UI.

**Comment by ssilvert** [comment]
Hi @Alexcei88.  Thanks for the feedback.  We just added those screens this week and fine-grained permissions will be completed soon.  Fine-grained permissions will be in the next release of the new admin UI.

If you find other bugs or omissions, please report them here: https://github.com/keycloak/keycloak-admin-ui/issues

**Reply by mattgagliardi** [reply]
Thank you for taking on this addition, we're mid-stream on moving from WildFly over to JVM Quarkus on our way to native Quarkus.  This is an important feature for us.
